src/Analyzers/CSharp/Analyzers/CSharpAnalyzers.projitems

```diff
@@ -55,4 +55,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)UsePatternMatching\CSharpIsAndCastCheckDiagnosticAnalyzer.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)UseThrowExpression\CSharpUseThrowExpressionDiagnosticAnalyzer.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(DefaultLanguageSourceExtension)' != ''">
```

Comment:
When is DefaultLanguageSourceExtension not set?

Reply:
I think it occurred during the evaluation of the **shproj** file.

Reply:
OK, odd.